docs/userGuide/syntax/questions.mbdf

```diff
@@ -1,130 +1,343 @@
-## Questions
+## Questions and Quizzes
 
-**Question component consists of a question body, a hint and an answer.**
+Question and quiz components provide an easy way to test readers on the relevant content topic in the page.
 
-<include src="codeAndOutput.md" boilerplate >
+#### Introduction
+
+Question components (`<question>`) can be one of the following types: **Mcq**, **Checkbox** or **Text**.
+
+In all cases, note that content directly inserted in between `<question>...</question>` will be inserted into the **question body**.
+
+You can also insert markdown into the **header** or **hint box**, by using the `header` and `hint` attributes respectively. Click the hint button below to see how the hint box turns out!
+
+<include src="codeAndOutputSeparate.md" boilerplate >
 <variable name="highlightStyle">html</variable>
+<variable name="heading">Header and hint syntax</variable>
 <variable name="code">
-<question>
-  The question body. <md>:grey_question:</md>
-  <div slot="hint">
-    Question hint. <md>:crystal_ball:</md>
-  </div>
-  <div slot="answer">
-    Question answer. <md>:pencil:</md>
-  </div>
+<!-- Insert markdown into the header and hint using the respective attributes -->
+<question ... header="Which of the following is correct? (**Header**)" hint="Think out of the box! :fas-box:">
```

Comment:
Should this be an mcq question instead?

Reply:
What's this `...` here?

Reply:
I made this checkbox for the purposes of the later more elaborate example with slots (`Challenge: Try to get all the options correct in your first try!`).

changed `...` -> `type="checkbox"`

Reply:
In that question, the question itself can be rephrased to fit a check-box type. The current question is an equation that can only have one answer. Hence, the reader will wonder why we are using check boxes.

Reply:
not sure I follow, isn't the snippet above the same question with two answers?:

![Untitled](https://user-images.githubusercontent.com/3306138/88474829-f6a26480-cf5c-11ea-8816-0ee77ec447b6.png)

Reply:
My bad. I didn't read the question closely. In fact, I read the last line only and thought it's a simple addition question for primary kids.